torch/csrc/autograd/python_variable.cpp

```diff
@@ -1697,6 +1698,12 @@ void concrete_dispatch_fn(
   // overload resolution but is more complicated (need to expose separate
   // functions per overload)
   py::handle torch_api_function = py::module::import("torch").attr("ops").attr(ns).attr(func_name);
+  py::handle torch_api_function_overload;
```

Comment:
You should update the comment above that begins with "For now, overloads get coalesced."